go.sum

```diff
@@ -206,6 +207,8 @@ github.com/julienschmidt/httprouter v1.2.0/go.mod h1:SYymIcj16QtmaHHD7aYtjjsJG7V
 github.com/julienschmidt/httprouter v1.3.0/go.mod h1:JR6WtHb+2LUe8TCKY3cZOxFyyO8IZAc4RVcycCCAKdM=
 github.com/kanopy-platform/k8s-auth-portal v0.1.0 h1:qKoTx50g8FkWgA4DqXz0KOU4F5Z3w5mDHbN0mGgzT3E=
 github.com/kanopy-platform/k8s-auth-portal v0.1.0/go.mod h1:UGHaJELJTCu04hKPQwZvI3FoULB2XnJ0WHDasesNAAs=
```

Comment:
nit: I think if you run `go mod tidy` again, these should get removed

Reply:
done